src/main/resources/explanations/challenge29.adoc

```diff
@@ -0,0 +1,6 @@
+=== The Client Secret
+
+A user enters some data in some mandatory field in a form, Now he/she didn't read the policies carefully and now his/her secret
```

Comment:
Can you rewrite this in something along the lines of : the client requires a random value of the server as a secret. Given it is random and not shown at startup you cannot "eavesdrop" it at the server. But what if we store it wrongly at the client?